config/jobs/kubernetes-sigs/image-builder/image-builder-presubmits.yaml

```diff
@@ -0,0 +1,23 @@
+presubmits:
+  kubernetes-sigs/image-builder:
+  - name: pull-azure-all
+    labels:
+      preset-azure-cred: "true"
+    always_run: false
```

Comment:
Assuming the intent is that this will get flipped to true after https://github.com/kubernetes-sigs/image-builder/pull/313 merges?

Reply:
Do we need to worry about setting max_concurrency for this job, or is it relatively safe to kick these off any number of times concurrently?

Reply:
> Assuming the intent is that this will get flipped to true after kubernetes-sigs/image-builder#313 merges?

correct, for now I want to use this to test kubernetes-sigs/image-builder#313 without running it on other PRs

> Do we need to worry about setting max_concurrency for this job, or is it relatively safe to kick these off any number of times concurrently?

the limit is the number of Azure resources it consumes but it's only 1 vm per job so relatively low cost. I can set a max concurrency just to be safe.

Reply:
Added concurrency